src/integTest/resources/legacy-syntax/print_banner_true.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
```

Comment:
This is too similar to [control_default.gradle](https://github.com/sonatype-nexus-community/scan-gradle-plugin/blob/da12fded567579249f360fa2000374da556f49da/src/integTest/resources/legacy-syntax/control_default.gradle)

Reply:
> This is too similar to [control_default.gradle](https://github.com/sonatype-nexus-community/scan-gradle-plugin/blob/da12fded567579249f360fa2000374da556f49da/src/integTest/resources/legacy-syntax/control_default.gradle)

Removed in 22c5d6d2